schedule/yast/yast2_ncurses_gnome.yaml

```diff
@@ -0,0 +1,49 @@
+---
+name:           yast2_ncurses_gnome
+description:    >
+    Test for yast2 UI, ncurses only. Running on created gnome images
+    which provides both text console for ncurses UI tests as well as
+    the gnome environment for the GUI tests.
+schedule:
+    - "{{uefi_boot}}"
+    - "{{bootloader_start}}"
+    - boot/boot_to_desktop
+    - console/prepare_test_data
+    - console/consoletest_setup
+    - "{{all_modules}}"
+    - "{{modules_from_default_installation}}"
+conditional_schedule:
+    uefi_boot:
+        ARCH:
+            aarch64:
+            - boot/uefi_bootmenu
+    bootloader_start:
+        ARCH:
+            s390x:
+            - installation/bootloader_start
+    all_modules:
+        ARCH:
+            x86_64:
+                - console/yast2_rmt
+                - console/yast2_ntpclient
+                - console/yast2_tftp
+                - console/yast2_proxy
+                - console/yast2_vnc
+                - console/yast2_nis
+                - console/yast2_http
+                - console/yast2_ftp
+                - console/yast2_apparmor
+                - console/yast2_lan
+                - console/yast2_lan_hostname
+                - console/yast2_dns_server
+                - console/yast2_nfs_client
+                - console/yast2_snapper_ncurses
+                - console/coredump_collect
+    modules_from_default_installation:
```

Comment:
Why do we need this second condition? We could combine it with the one above as it doesn't bring additional value.
Running `- console/coredump_collect` actually makes sense for all cases.

Reply:
I can put out of the condition that last module.
The problem is the order of execution, I didn't want to change that, should I? I thought those tests might behave funny if I arrange yast2_lan in different place as I don't consider those ui tests really robust. wdyt?

Reply:
Yeah, changing the order might break tests, but you will keep it same for 64bit, so there is no change there. So I don't see how that's related to having 2 conditions instead of 1? Or am I missing something?

Reply:
fixed, my bad! I was confusing conditions with condional schedule sections, they can be put all together as they don't overlap, good catch.